index.html

```diff
@@ -9,261 +9,392 @@
+             </div>
+             
+         </header>
+         <section class="projects">
+             <h3 class="title">Projects</h3>
+             <div class="projects-wrapper">
+                 <div class="project-image">
+                     <img class="project-img" src="images/Rectangle1.png" alt="image">
+                 </div>
+                 <div class="project-lg">
+                   <div class="project-lg-1">
+                     <h3>Project name goes here</h3> 
+                     
+                     <div class="list-languages">
+                         <ul>
+                             <li><span>HTML/CSS</span></li>
+                             <li><span>Ruby on Rails</span></li>
+                             <li><span>JavaScript</span></li>
+                         </ul>
+                     </div>
+                     <div class="project-view">
+                             <span class="project-view-union cursor">See this project</span>
+                             
+                             <span><img src="images/Union.png" alt="union"></span>
+                     </div>
+                   </div>
+                 </div>
+             </div>
+             <div class="projects-wrapper">
+                 <div class="project-image">
+                     <img class="project-img" src="images/Rectangle2.png" alt="image">
+                 </div>
+                 <div class="project-lg">
+                   <div class="project-lg-1">
+                     <h3>Project name goes here</h3> 
+                     
+                     <div class="list-languages">
+                         <ul>
+                             <li><span>HTML/CSS</span></li>
+                             <li><span>Ruby on Rails</span></li>
+                             <li><span>JavaScript</span></li>
+                         </ul>
+                     </div>
+                     <div class="project-view">
+                             <span class="project-view-union cursor">See this project</span>
+                             
+                             <span><img src="images/Union.png" alt="union"></span>
+                     </div>
+                   </div>
+                 </div>
+             </div>
+             <div class="projects-wrapper">
+                 <div class="project-image">
+                     <img class="project-img" src="images/Rectangle3.png" alt="image">
+                 </div>
+                 <div class="project-lg">
+                   <div class="project-lg-1">
+                     <h3>Project name goes here</h3> 
+                     
+                     <div class="list-languages">
+                         <ul>
+                             <li><span>HTML/CSS</span></li>
+                             <li><span>Ruby on Rails</span></li>
+                             <li><span>JavaScript</span></li>
+                         </ul>
```

Comment:
- [ ] In mobile width the buttons still don't have paddings, Please add paddings in mobile view as well 😉
<img width="316" alt="Screenshot 2022-02-25 at 13 07 19" src="https://user-images.githubusercontent.com/77038610/155687231-2dec202f-558f-46dc-8253-e97cc3e47b3f.png">